core/client/src/test/java/alluxio/client/file/options/CreateDirectoryOptionsTest.java

```diff
@@ -11,19 +11,26 @@
 
 package alluxio.client.file.options;
 
+import alluxio.CommonTestUtils;
 import alluxio.Constants;
 import alluxio.client.ClientContext;
+import alluxio.client.UnderStorageType;
 import alluxio.client.WriteType;
 import alluxio.thrift.CreateDirectoryTOptions;
 
 import org.junit.Assert;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
 
 import java.util.Random;
 
 /**
  * Tests for the {@link CreateDirectoryOptions} class.
  */
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(UnderStorageType.class)
```

Comment:
would using CreateDirectoryOptions.class here instead work? if not, I would add a comment here to explain why we need to do this ... similarly for other tests in this PR